CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## Unreleased changes
 
 - Add `cis3` module and `Cis3Contract` for interacting with CIS3 contracts.
+- Updated the concordium-base submodule to incorporate protocol 7 changes.
+  This updates some protobuf messages relating to cooldown and baker pool status.
```

Comment:
I think we need more detail on the public types which have changed, it should be helping the user in how to migrate to this version.

Reply:
I added some more detail! :)

Reply:
Excellent :+1: